flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -397,7 +398,7 @@ class TableEnvironmentTest {
       () => tableEnv.executeSql("SELECT c FROM my_view /*+ OPTIONS('is-bounded' = 'true') */"))
       .hasMessageContaining("View '`default_catalog`.`default_database`.`my_view`' " +
         "cannot be enriched with new options. Hints can only be applied to tables.")
-      .isInstanceOf(classOf[ValidationException])
+      .isInstanceOf[ValidationException]
```

Comment:
`isInstanceOf` is the method of `AbstractAssert` instead of `scala`, please revert the change